http/src/main/resources/reference.conf

```diff
@@ -442,6 +442,21 @@ contentSecurityPolicy {
   ]
 }
 
+refererConfig {
+  # list of valid hosts is environment specific so it's overriden in firecloud develop - these are the valid
+  # hosts for our dev environment (including fiab)
+  validHosts = [
+    "localhost:3000",
+    "localhost:4200",
+    "localhost:443",
+    "bvdp-saturn-dev.appspot.com",
+    "all-of-us-workbench-test.appspot.com",
+    "dev.terra.biodatacatalyst.nhlbi.nih.gov",
+    "leonardo-fiab.dsde-dev.broadinstitute.org",
```

Comment:
I think this should be `leonardo-fiab.dsde-dev.broadinstitute.org:30443`? 

(Since fiabs have a port)

Reply:
going to retest in my fiab by running the hotswap script since fiab-start keeps failing